serverside/zerotrace/example/main.go

```diff
@@ -103,6 +147,13 @@ func webSocketHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	var appLayerRtt = calcStats(ms)
+
+	// Run ICMP measurement towards the clientIP on network layer
+	icmpResults, err := icmpPinger(clientIP)
+	if err != nil {
+		l.Println("ICMP Ping Error: ", err)
+	}
+	nwLayerRttICMP = icmpResults.MinRtt
 	done := make(chan bool)
 	// Start 0trace measurement in the background.
 	go func() {
```

Comment:
The ICMP pinger blocks until it's finished. Are we sure that the WebSocket connection is still active by the time we run 0trace?

Reply:
Included it in [5c7d0c2](https://github.com/reethikar/zerotrace-eval/pull/4/commits/5c7d0c25ed499e2f295de72eb5fab4c251278cd3)